Description: uuid.t: skip when Data::UUID is unavailable during testing
 Otherwise the tests require Module::Build's _build/ directory which breaks
 Debian continuous integration tests.
 .
 The new test uses the same logic that is used in Object::ID itself when
 creating the object_uuid method.
Author: Damyan Ivanov <dmn@debian.org>
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=105123
Bug: https://rt.cpan.org/Ticket/Display.html?id=105123

--- a/t/uuid.t
+++ b/t/uuid.t
@@ -5,12 +5,10 @@
 use strict;
 use warnings;
 
-use Module::Build;
 use Test::More;
 
-plan skip_all => "Data::UUID not available for object_uuid" unless do {
-    my $build = Module::Build->resume;
-    $build->feature("object_uuid");
+plan skip_all => "Data::UUID not available for object_uuid" unless eval {
+    require Data::UUID;
 };
 
 {
